config/config.properties

```diff
@@ -106,6 +106,47 @@ pravegaservice.dataLog.impl.name=BOOKKEEPER
 # pravegaservice.storage.layout=ROLLING_STORAGE
 # NOTE: ChunkedSegmentStorage is an experimental feature. Its behavior and/or API should be expected to be in flux until fully released.
 
+# Size of chunk in bytes above which it is no longer considered a small object.
+# For small source objects, concat is not used.
+# Valid values: non-negative long
+# Default value: 0
+# storage.concat.size.bytes.min=0
+
+# Size of chunk in bytes above which it is no longer considered for concat.
+# Valid values: non-negative long
+# Default value: 9223372036854775807
+# storage.concat.size.bytes.max=9223372036854775807
+
+# Maximum size for the buffer used while copying of data from one chunk to other.
+# Valid values: non-negative integer
+# Default value: 1048576 (1 MB)
+# storage.appends.buffer.size.bytes.max=1048576
+
+# Max number of indexed segments to keep in read cache.
+# Valid values: non-negative integer
+# Default value: 1024 (1 K)
+# storage.readindex.segments.max=1024
+
+# Max number of indexed chunks to keep per segment in read cache.
+# Valid values: non-negative integer
+# Default value: 1024 (1 K)
+# storage.readindex.chunksPerSegment.max=1024
+
+# Max number of indexed chunks to keep in cache.
+# Valid values: non-negative integer
+# Default value: 16384 (16 K)
+# storage.readindex.chunks.max=16384
+
+# The maximum size of a single Segment Chunk in Storage for metadata segments.
+# Valid values: non-negative long
```

Comment:
Please be explicit about the bounds. A long can be a signed long or unsigned long. In this case, if I remember, the upper bound is 2^62, isn't it?

Please fix this for the other config descriptions too.

Reply:
done